transaction/src/action/delegate.rs

```diff
@@ -31,6 +33,10 @@ impl IsAction for Delegate {
     fn balance_commitment(&self) -> penumbra_crypto::balance::Commitment {
         self.balance().commit(Fr::zero())
     }
+
+    fn decrypt_with_perspective(&self, txp: &TransactionPerspective) -> anyhow::Result<ActionView> {
```

Comment:
Now that we have the design decision to have an "interpreted" `ActionView`, I think that this method would really be better named `view_from_perspective`, rather than `decrypt` -- and we won't need a `Result`, since viewing is infallible.

Reply:
I suggest we leave the signature for the moment & make an issue for next release to go over this work and make sure Results/Errors arising from composed function calls are handled in a way that's informative, because we can make `view_from_perspective` infallible, but there are component functions that may fail & I don't want to refactor around that thoughtlessly.